BUILDING.md

```diff
@@ -1,244 +1,104 @@
-# Building Bottlerocket
+# How to build the Bottlerocket core kit
 
-If you'd like to build your own image instead of relying on an Amazon-provided image, follow these steps.
-You can skip to the [setup guide for Kubernetes](QUICKSTART-EKS.md) or the [setup guide for Amazon ECS](QUICKSTART-ECS.md) to use an existing image in Amazon EC2.
-(We're still working on other use cases!)
-
-## Build an image
-
-### Dependencies
+If you'd like to build your own copy of the core kit for local development, follow these steps.
 
+## Dependencies
 #### System Requirements
-
 The build process artifacts and resulting images can consume in excess of 80GB in the local directory.
-
-The build process is also fairly demanding on your CPU, since we build all included software from scratch.
-(The first time.  Package builds are cached, and only changes are built afterward.)
+The build process is also fairly demanding on your CPU, since we build all included software from scratch. This is only done the first time. Package builds are cached, and only changes are built afterward.
 The build scales well to 32+ cores.
-The first time you build, the fastest machines can take about 12 minutes while slower machines with only a couple cores can take 3-4 hours.
-
+The first time you build, the fastest machines can take about 20 minutes while slower machines with only a couple cores can take 3-4 hours.
 #### Linux
-
 The build system requires certain operating system packages to be installed.
-
-Ensure the following OS packages are installed:
-
+Ensure the following packages are installed:
 ##### Ubuntu
-
 ```shell
 apt install build-essential openssl libssl-dev pkg-config liblz4-tool
 ```
-
 ##### Fedora
-
 ```shell
 yum install make automake gcc openssl openssl-devel pkg-config lz4 perl-FindBin perl-lib
 ```
 
-
 #### Rust
-
 The build system is based on the Rust language.
 We recommend you install the latest stable Rust using [rustup](https://rustup.rs/), either from the official site or your development host's package manager.
 Rust 1.51.0 or higher is required.
-
 To organize build tasks, we use [cargo-make](https://sagiegurari.github.io/cargo-make/).
 To get it, run:
-
 ```shell
 cargo install cargo-make
 ```
+There are two paths to enable the required capabilities in docker for use with OOTB. You can modify your existing docker daemon to enable OCI support or you can install an optional container orchestration tool called Crane.
 
-#### Docker
-
-Bottlerocket uses [Docker](https://docs.docker.com/install/#supported-platforms) to orchestrate package and image builds.
+We recommend using `crane` (and `gcrane`) over `docker` as it has shown better performance in our testing.
```

Comment:
I said `gcrane` before, but it's actually `krane` that has the built-in support for cloud provider credentials, and that's what I run locally.